### PR TITLE
RTEMIS-183: Defined the schema for the School seat status table.

### PR DESCRIPTION
**Pull request by RohanSingh-inno**
Ticket # : https://innoraft.atlassian.net/browse/RTEMIS-183

---
### What the PR does
`(Explain fixes, changes and impacts)`
- Defined the schema.

### CODE REVIEW

File: rte_mis.install [hunk 1]
```diff
@@ -40,3 +40,47 @@ function rte_mis_install() {
   }
 
 }
+
+/**
+ * Impelments hook_schema().
+ */
+function rte_mis_schema() {
+  $schema['lottery_school_seats_status'] = [
+    'description' => 'Stores the result of the lottery',
```
Comment (jaswinsingh) [inline rte_mis.install:49]: This will store the seat status for school, not the result of lottery.

File: rte_mis.install [hunk 2]
```diff
@@ -40,3 +40,47 @@ function rte_mis_install() {
...
+      $column_name = $value . ' seats';
+      $schema['lottery_school_seats_status']['fields'][$column_name] = [
+        'description' => 'Total available seats for' . $key,
+        'type' => 'int',
+        'unsigned' => TRUE,
+        'not null' => TRUE,
+        'default' => 0,
+      ];
+    }
+  }
```
Comment (jaswinsingh) [inline rte_mis.install:83]: - The column name should be the machine name of the language, not the label ie Hindi/English. Also, remove the "seats" from the column name.
- Change the description to Total RTE seat available for that language.

File: rte_mis.install (hunk 1 above)
Comment (RohanSingh-inno) [reply rte_mis.install]: ok

File: rte_mis.install (hunk 2 above)
Comment (RohanSingh-inno) [reply rte_mis.install]: sure

**Review by jaswinsingh — APPROVED** [review]

File: rte_mis.install [hunk 3]
```diff
@@ -40,3 +40,47 @@ function rte_mis_install() {
...
+      'school_name' => [
+        'description' => 'School Name',
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => TRUE,
+      ],
+      'school_id' => [
+        'description' => 'School Id',
+        'type' => 'int',
+        'unsigned' => TRUE,
```
Comment (jaswinsingh) [inline rte_mis.install:60]: @RohanSingh-inno Why have we created table `lottery_school_seats_status`  at the profile level?

**Pull request merged**